<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interactive Brokers Data Importer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="ibFeedPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>431</height>
            </rect>
           </property>
           <attribute name="label">
            <string>IB Feed</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Host Name:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="ibHostLineEdit">
                <property name="text">
                 <string>127.0.0.1</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Normal</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Port:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="ibPortSpinBox">
                <property name="maximum">
                 <number>9999</number>
                </property>
                <property name="value">
                 <number>7497</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="loginButton">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="logoutButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Logout</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="reconnectCheckbox">
              <property name="text">
               <string>Reconnect On Failure</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>128</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="databasePage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>431</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Database</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QCheckBox" name="mySqlCheckBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>MySql</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="hdf5CheckBox">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Use HDF5</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="mysqlGroupBox">
              <property name="title">
               <string>MySql</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QFormLayout" name="formLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Host Name:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="sqlHostNameLineEdit">
                   <property name="text">
                    <string>127.0.0.1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Port:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSpinBox" name="sqlPortSpinBox">
                   <property name="maximum">
                    <number>99999</number>
                   </property>
                   <property name="value">
                    <number>3306</number>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>User Name:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="sqlUserNameLineEdit">
                   <property name="text">
                    <string>root</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Password:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="sqlPasswordLineEdit">
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Symbol Database Name:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QLineEdit" name="sqlDatabaseNameLineEdit">
                   <property name="text">
                    <string>StockIntradayData</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Symbol Table Name:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QLineEdit" name="sqlSymbolTableNameLineEdit">
                   <property name="text">
                    <string>stock_companyInfo</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Output Database Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QLineEdit" name="sqlOutputDatabaseLineEdit"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="loadSymbolTablePushButton">
                 <property name="text">
                  <string>Load Symbol Table</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="hdf5GroupBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="title">
               <string>HDF5</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Output Folder</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="hdf5OutputFolderLineEdit"/>
                 </item>
                 <item>
                  <widget class="QToolButton" name="hdf5OutputFileToolButton">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>45</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="downloadPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>431</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Download</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Time Frame:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="timeFrameComboBox">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>S1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>S5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>S15</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>S30</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>M1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>M2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>M3</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>M5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>M15</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>M30</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>H1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>D1</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Number of Months:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="numberOfMonthsSpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>24</number>
                </property>
                <property name="value">
                 <number>6</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="barsToLoadCheckbox">
                <property name="text">
                 <string>Bars To Load</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="datewiseCheckbox">
                <property name="text">
                 <string>Date Wise</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="fromGroupbox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="title">
               <string>From</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QDateEdit" name="fromDateEdit">
                 <property name="minimumDate">
                  <date>
                   <year>1752</year>
                   <month>11</month>
                   <day>14</day>
                  </date>
                 </property>
                 <property name="displayFormat">
                  <string>M/yy</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="fromTimeEdit"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="toGroupbox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="title">
               <string>To</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QDateEdit" name="toDateEdit"/>
               </item>
               <item>
                <widget class="QTimeEdit" name="toTimeEdit"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="startButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stopButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="realtimeDataCheckbox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Enable Realtime Data</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>207</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
